tsconfig.json

```diff
@@ -19,7 +18,9 @@
     "experimentalDecorators": true,
     "skipLibCheck": true,
     "declaration": true,
-    "declarationDir": "/lib"
+    "emitDeclarationOnly": true
   },
-  "exclude": []
+  "exclude": [
```

Comment:
This one is added so typings from example files are not generated. In the next PRs, it may be beneficial to extract example out of the src to be able to typecheck it too by a separate tsconfig.json but it's rather not really important.